### ISSUE TITLE
[F2-220] Alternative Installation Method

### ISSUE THREAD

**Issue by zenekron** [cskau/TM-F2-420-Linux#1]
TLDR: I installed Arch through SSH using a custom SSH-enabled live ISO built with [archiso].

#### Premise
I recently installed Arch Linux on a TerraMaster F2-220. While the installation process documented in the readme has been a great help in doing so, I took some fundamentally different choices which led to what I think is a simpler installation method.

### Building the live ISO
Following the instruction on the [archiso page][archiso] create a new image off the `releng` profile and [configure](https://wiki.archlinux.org/index.php/Archiso#Configure_the_live_medium) it to automatically do what's described in [Install from SSH](https://wiki.archlinux.org/index.php/Install_from_SSH) at init.  
Generally you'll end up appending to `airootfs/root/customize_airootfs.sh` something like this:
```sh
# set PermitRootLogin to yes
sed -i 's/#\(PermitRootLogin \).\+/\1yes/' /etc/ssh/sshd_config

# non-interactive passwd
echo 'root:toor' | chpasswd

# Enable OpenSSH daemon
systemctl enable sshd.service
```

Finally [build the ISO](https://wiki.archlinux.org/index.php/Archiso#Build_the_ISO) and [burn it onto an USB stick](https://wiki.archlinux.org/index.php/USB_flash_installation_media#BIOS_and_UEFI_bootable_USB).

### Booting from USB
Plug the USB stick in one of the USB ports on the back, what happens next strictly depends on the boot 
order: if TOS boots up, you'll need to open the NAS and remove the internal USB drive, otherwise you're good to go (unless you plan to change the internal drive anyway).

## Installation
Once you SSH into your NAS the installation process is almost the same as that of any other machine: just follow the official [Installation Guide](https://wiki.archlinux.org/index.php/Installation_guide).

#### Differences from a standard installation
Considering that your NAS will most likely be a headless machine, you should be particularly careful when setting up networking and ssh access.  

### Boot Loader
One thing I found out is that the motherboard supports UEFI, As such I decided to go with `systemd-boot` as a boot loader and it worked flawlessly so far.

[archiso]: https://wiki.archlinux.org/index.php/Archiso
